package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/algolia/autocomplete.js.git"
   },
   "dependencies": {
-    "npm-zepto": "^1.1.7"
+    "zepto": "^1.0.1"
```

Comment:
@vvo `1.0.1` is super old; I wouldn't go for that one. Rather copy it locally then; what do you think?